### ISSUE TITLE
Problems plotting

### ISSUE THREAD

**Issue by hgrecco** [hgrecco/pint-pandas#43]
```python

import pandas as pd
import pint

ureg = pint.UnitRegistry()

df = pd.DataFrame({
    "torque": pd.Series([1, 2, 2, 3], dtype="pint[lbf ft]"),
    "angular_velocity": pd.Series([1, 2, 2, 3], dtype="pint[rpm]"),
})
print(df)

df.plot()
```
yields:
```
Traceback (most recent call last):
  File "/Users/grecco/Documents/code/simbio/examples/test_plot.py", line 13, in <module>
    df.plot()
  File "/Users/grecco/anaconda3/envs/sci38/lib/python3.8/site-packages/pandas/plotting/_core.py", line 847, in __call__
    return plot_backend.plot(data, kind=kind, **kwargs)
  File "/Users/grecco/anaconda3/envs/sci38/lib/python3.8/site-packages/pandas/plotting/_matplotlib/__init__.py", line 61, in plot
    plot_obj.generate()
  File "/Users/grecco/anaconda3/envs/sci38/lib/python3.8/site-packages/pandas/plotting/_matplotlib/core.py", line 261, in generate
    self._compute_plot_data()
  File "/Users/grecco/anaconda3/envs/sci38/lib/python3.8/site-packages/pandas/plotting/_matplotlib/core.py", line 410, in _compute_plot_data
    raise TypeError("no numeric data to plot")
TypeError: no numeric data to plot
```

Any ideas how to fix this without monkey patching pandas? Any way to declare what is a "numeric type"?

**Comment by andrewgsavage** [comment]
No. A workaround is to do `df.pint.dequantify().plot()` but won't get give matplotlib units support.

This is the pandas issue https://github.com/pandas-dev/pandas/issues/26173

**Comment by andrewgsavage** [comment]
closed by #116
